src/main/java/org/folio/orders/utils/RequestContextUtil.java

```diff
@@ -10,6 +10,9 @@ private RequestContextUtil() {
   }
 
   public static RequestContext createContextWithNewTenantId(RequestContext requestContext, String tenantId) {
+    if (tenantId == null) {
```

Comment:
better to use StringUtils

Reply:
RMB is written just with null check, so I am okay to leave as is also with null check. It's personally my preference to use StringUtils in every place, but we can leave as is for current place.
![image](https://github.com/folio-org/mod-orders/assets/25097693/aa2d5f03-2625-42e4-837c-ba4312a20698)